setup.py

```diff
@@ -59,6 +59,11 @@
 ]
 
 install_requires = [
+    "invenio-db>=1.0.0",
+    "invenio-search>=1.2.0",
```

Comment:
The `invenio-search` dependency should be removed from here, since `invenio-search` will be installed using the extras.